sdk/modelsrepository/azure-iot-modelsrepository/azure/iot/modelsrepository/_version.py

```diff
@@ -0,0 +1,8 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
```

Comment:
Lets also append a 1.0.0b2 changelog entry to `CHANGELOG.md` with the new version

Example:
```
# Release History

## 1.0.0b2 (unreleased)


## 1.0.0b1 (2021-04-27)

* Initial (Preview) Release
```